### PR TITLE
close #127: clean up remnants of SDF 1.0/1.1 in .md and .cddl

### LINKED ISSUE #127: Fully clean up remnants of SDF 1.0/1.1 (in CDDL, too)

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -1124,17 +1122,16 @@ present specification.
...
+| contentFormat | string                                      | content type (IANA media type string plus parameters), encoding (note 2) | N/A     |
 | sdfType       | string ({{sdftype}})                        | sdfType enumeration (extensible)                                         | N/A     |
 | sdfChoice     | named set of data qualities ({{sdfchoice}}) | named alternatives                                                       | N/A     |
 | enum          | array of strings                            | abbreviation for string-valued named alternatives                        | N/A     |
 {: #sdfdataqual2 title="SDF-defined Qualities of sdfData"}
 
 
-1. Note that the quality `unit` was called `units` in SDF draft 1.0.
+1. Note that the quality `unit` was called `units` in earlier drafts
+   of SDF.
```
Comment (akeranen) [inline sdf.md:1134]: Would “earlier drafts of this specification” work better than “drafts of SDF”?

**Review by akeranen — APPROVED** [review]
Based on reading the changes LGTM but should also test the new CDDL/JSO to make sure it works as expected

**Comment by cabo** [comment]
> Based on reading the changes LGTM but should also test the new CDDL/JSO to make sure it works as expected

I tested the CDDL against the playground (and a failing model as well).
Need to remember how to test the JSO...

**Review by JKRhb — APPROVED** [review]
LGTM!

File: sdf.md (hunk 1 above)
Comment (cabo) [reply sdf.md:1134]: > Would “earlier drafts of this specification” work better than “drafts of SDF”?

Ja.  Fix s.u.

**Pull request merged**